pkg/specs/podmonitor.go

```diff
@@ -42,9 +44,18 @@ func (c ClusterPodMonitorManager) IsPodMonitorEnabled() bool {
 
 // BuildPodMonitor builds a new PodMonitor object
 func (c ClusterPodMonitorManager) BuildPodMonitor() *monitoringv1.PodMonitor {
+	version, _ := c.cluster.GetPostgresqlMajorVersion()
+
 	meta := metav1.ObjectMeta{
 		Namespace: c.cluster.Namespace,
 		Name:      c.cluster.Name,
+		Labels: map[string]string{
+			utils.ManagedByLabelName: utils.ManagerName,
+			utils.AppLabelName:       utils.AppName,
+			utils.InstanceLabelName:  c.cluster.Name,
```

Comment:
I find this a bit discontinuous.
I mean, in the K8s page on recommended labels, you see: application -> mysql, instance -> mysql-abcxyz
Instead here, we'd have application -> postgres instance -> acme-corp-cluster
Maybe something like "postgres-<cluster-name>"?

Reply:
the application and instance are necessarily linked, the instance is just the unique name of that entity grouping.

i.e. in the context a helm app.kubernetes.io/name is usually the name of the chart, while app.kuberentes.io/instance is usually the name of the release

i can deploy cloudnative-pg chart twice and the first one i use the release name database-operator, while the second one i name cnpg, the name will be cloudnative-pg for both but the instances will be database-operator and cnpg resepctively